js/webview/swipeEvents.js

```diff
@@ -82,21 +88,20 @@ window.addEventListener('wheel', function (e) {
 
       e.preventDefault()
     }
-    platformZoomKey = e.metaKey
   }
   /* cmd-key while scrolling should zoom in and out */
 
-  if (verticalMouseMove > 55 && platformZoomKey) {
+  if (verticalMouseMove > 55 && platformZoomKey && initialZoomKeyState) {
     verticalMouseMove = -10
     zoomOut()
   }
 
-  if (verticalMouseMove < -55 && platformZoomKey) {
+  if (verticalMouseMove < -55 && platformZoomKey && initialZoomKeyState) {
     verticalMouseMove = -10
     zoomIn()
   }
 
-  if (platformZoomKey) {
+  if (platformZoomKey && initialZoomKeyState) {
     e.preventDefault()
   }
```

Comment:
Not directly related to this PR, but these could be rewritten as:

```js
if (platformZoomKey && initialZoomKeyState) {
  if (verticalMouseMove > 55) {
    verticalMouseMove = -10
    zoomOut()
  }

  if (verticalMouseMove < -55) {
    verticalMouseMove = -10
    zoomIn()
  }

  e.preventDefault()
}
```

so `platformZoomKey && initialZoomKeyState` wouldn't be duplicated anymore.

Reply:
Fixed in 31088da3bd5227cd4bab0358751589c9c64aae39.